tests/queries/0_stateless/01941_multiple_pipe_read.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
```

Comment:
You forget to make it eXecutable (hence fasttest fails)

Reply:
Fixed that